app/models/waste_carriers_engine/registration.rb

```diff
@@ -110,25 +110,21 @@ def original_activation_date
     end
 
     def increment_certificate_version(user = nil)
-      new_version = metaData.certificateVersion.to_i.succ
-      metaData.update(certificateVersion: new_version)
+      # If history is empty, this is the first version so keep the default value
+      new_version = metaData.certificateVersionHistory.empty? ? 1 : metaData.certificateVersion.to_i.succ
```

Comment:
Shouldn't certificateVersion be 1 by default ? Why does it hardcode 1 instead of taking the current certificateVersion value?

Reply:
Yes, it should be 1 (the default value) if version history is empty. The code just worked a little more cleanly this way, skipping an update of 1 with 1.